src/ui_fsmenu/about.cc

```diff
@@ -68,8 +68,7 @@ About::About(MainMenu& fsmm, UI::UniqueWindow::Registry& r)
 			std::string label, localized_label, value, localized_value;
 		};
 		const std::vector<ContentT> content = {
-		   {"Version:", _("Version:"), format("%1$s (%2$s)", build_id(), build_type()),
-		    format(_("%1$s (%2$s)"), build_id(), build_type())},
+		   {"Version:", _("Version:"), build_ver_details(), format(_("%1$s"), build_ver_details())},
```

Comment:
The second `format` call is also redundant, since there is no reason why anyone would translate this as something different. You can leave the last argument empty if no custom localization is needed.
```suggestion
		   {"Version:", _("Version:"), build_ver_details(), ""},
```